app/support/order_uncanceler.rb

```diff
@@ -31,10 +31,10 @@ def uncancel_to_complete(order_detail)
 
         order_detail.assign_price_policy(fulfilled_at)
         order_detail.save!
-        puts "#{order_detail} was uncanceled"
+        Rails.logger.info "OrderDetail #{order_detail} was uncanceled"
         true
       rescue => e
-        puts "Could not save #{order_detail} because #{e.message}"
+        Rails.logger.info "Could not save OrderDetail #{order_detail} because #{e.message}"
         raise ActiveRecord::Rollback
       end
     end
```

Comment:
Good call, it is in a rake task.  Hmmm.  I could have the parent task set `Rails.logger = Logger.new(STDOUT)` so it still `puts` to the command line without getting `puts` during spec execution.  Do you have any better ideas?  Thanks!